sqe-api-server/RealtimeHubs/SubscriptionHub.cs

```
@@ -29,29 +54,36 @@ public partial class MainHub
         /// <returns></returns>
         public async Task SubscribeToEdition(uint editionId)
         {
-            var user = await _userService.GetCurrentUserObjectAsync(editionId);
+            try
+            {
+                var user = await _userService.GetCurrentUserObjectAsync(editionId);
 
-            if (!user.MayRead)
-                throw new StandardExceptions.NoReadPermissionsException(user);
+                if (!user.MayRead)
+                    throw new StandardExceptions.NoReadPermissionsException(user);
```

Isn't it better to throw an exception saying the edition does not exist? If the user has no permissions to it, it should be as if there's no edition at all.